Source/Core/DolphinQt/RenderWidget.cpp

```diff
@@ -51,7 +51,10 @@ RenderWidget::RenderWidget(QWidget* parent) : QWidget(parent)
     if (!Config::Get(Config::MAIN_RENDER_WINDOW_AUTOSIZE) || isFullScreen() || isMaximized())
       return;
 
-    resize(w, h);
+    const auto dpr =
+        QGuiApplication::screens()[QApplication::desktop()->screenNumber(this)]->devicePixelRatio();
```

Comment:
``QDesktopWidget`` is deprecated. Please use ``window()->windowHandle()->screen()`` to obtain ``QScreen`` instead.

Reply:
Oh I totally missed your comment, sorry. I didn't know this was deprecated. I suppose we should fix the other usages of it too?